leapp/libraries/stdlib/__init__.py

```diff
@@ -30,3 +40,45 @@ def call(args, split=True):
     if split:
         return r.splitlines()
     return r
+
+
+def _logging_handler(fd, buffer):
```

Comment:
Update callback function signature
```suggestion
def _logging_handler((fd, fd_type), buffer):
```

Also in the body of the function we should check `fd_type` and not `fd`

Reply:
@shaded-enmity ,  suggested change has been done
https://travis-ci.org/oamg/leapp/builds/492657740
Failing in python 3.6 env (SyntaxError: invalid syntax)
In 2.7 env it's ok

Reply:
solved